src/main/resources/bundleconfiguration/sscs-bundle-edited-config.yaml

```diff
@@ -10,13 +10,20 @@ sort:
 hasFolderCoversheets: false
 pageNumberFormat: numberOfPages
 documentNameValue: /documentFileName
+documentLinkValue: /editedDocumentLink
```

Comment:
What does this do?

Reply:
This is where we got lucky, by default it looks for the document url at property documentLink, this allows us to look at the property editedDocumentLink for the link for the edited document. If it didn't do this the model change wouldn't have worked.

Reply:
Ha ok